node/src/Pos/Explorer/Txp/Local.hs

```diff
@@ -177,8 +179,8 @@ eTxProcessTransactionNoLock itw@(txId, txAux) = reportTipMismatch $ runExceptT $
         -> HeaderHash
         -> (TxId, TxAux)
         -> Maybe Timestamp
-        -> (UtxoModifier, MemPool, UndoMap, HeaderHash, ExplorerExtra)
-        -> Either ToilVerFailure (UtxoModifier, MemPool, UndoMap, HeaderHash, ExplorerExtra)
+        -> ETxpLocalDataPure
+        -> Either ToilVerFailure ETxpLocalDataPure
```

Comment:
Bikeshedding: if we're only using this alias once, then I'm not sure it really buys us much. I'd omit this.